### PR TITLE
Changed request version to show providers and sign_in need now to add…

### PR DESCRIPTION
**Pull request by aljesusg**
… provider in header

### CODE REVIEW

File: spec/requests/v1/v1_apis_spec.rb [hunk 1]
```diff
@@ -12,15 +12,15 @@
         expect(response).to have_http_status(200)
         expect(response.content_type).to eq('application/json')
         expect(response.charset).to eq('utf-8')
-        expect(response.body).to eq("{\"data\":{\"version\":\"#{version}\"}}")
+        expect(response.body).to eq("{\"data\":{\"version\":\"1.0\",\"providers\":{\"github.com\":{\"type\":\"github\",\"enabled\":true,\"id_application\":null,\"server\":\"api.github.com\",\"version\":\"v3\",\"verify\":true}}}}")
       end
 
-      it 'gets the API version' do
+      it 'gets the API version2' do
```
Comment (chargio) [inline spec/requests/v1/v1_apis_spec.rb]: ?

Comment (aljesusg) [reply spec/requests/v1/v1_apis_spec.rb]: ^^ It's hard to make an spec with -e '<name of the test>' when all the tests have the same name ^^I'll change it

File: .gitignore [hunk 2]
```diff
@@ -7,6 +7,9 @@
 # Ignore bundler config.
 /.bundle
 
+# Ignore Idea
```
Comment (chargio) [inline .gitignore]: This should be in ~/.gitignore_global

File: spec/requests/v1/v1_apis_spec.rb [hunk 3]
```diff
@@ -12,15 +12,15 @@
...
         get '/v1/api/version'
         expect(response).to have_http_status(200)
         expect(response.content_type).to eq('application/json')
         expect(response.charset).to eq('utf-8')
-        expect(response.body).to eq("{\"data\":{\"version\":\"#{version}\"}}")
+        expect(response.body).to eq("{\"data\":{\"version\":\"1.0\",\"providers\":{\"github.com\":{\"type\":\"github\",\"enabled\":true,\"id_application\":null,\"server\":\"api.github.com\",\"version\":\"v3\",\"verify\":true}}}}")
```
Comment (chargio) [inline spec/requests/v1/v1_apis_spec.rb]: Could we add to the server address, the full login url built? In that way we can modifiy that in the backend

**Pull request merged**